subgraphs/level-finance/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "level-finance",
+  "license": "UNLICENSED",
+  "scripts": {
+    "prepare:constants": "mustache protocols/${npm_config_protocol}/config/deployments/${npm_config_id}/configurations.json src/common/constants.mustache > src/common/constants.ts"
```

Comment:
this can be removed. It is baked into `messari-subgraph-cli`

Reply:
@shashwatS22 I apologize, this is needed

Reply:
No worries, I reverted it.